tests/publiccloud/registration.pm

```diff
@@ -36,8 +38,9 @@ sub run {
 sub cleanup {
     my ($self) = @_;
     if (is_azure()) {
-        record_info('azuremetadata', $self->{run_args}->{my_instance}->run_ssh_command(cmd => "sudo /usr/bin/azuremetadata --api latest --subscriptionId --billingTag --attestedData --signature --xml"));
+        record_info('azuremetadata', $self->{my_instance}->run_ssh_command(cmd => "sudo /usr/bin/azuremetadata --api latest --subscriptionId --billingTag --attestedData --signature --xml"));
```

Comment:
I am pretty sure that if you dig into git history  you will find commit which is doing other way around ( switching to use `{my_instance}` from `{run_args}` :thinking: )  so we kinda running in circles here ... let's discuss this in a call

Reply:
OH, this is actually wrong. Let's have a call.